css/style.css

```diff
@@ -40,8 +41,9 @@ body {
 }
 
 .header__language {
-	margin-left: 5px;
-	padding-right: 42px;
+	margin-top: 10px;
+	margin-left: 0;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/LOCO_A/pullRequest?prid=6206892)